### ISSUE TITLE
Allow jump to the method specified by `TestCase`'s @dataprovider 

### ISSUE THREAD

**Issue by przepompownia** [phpactor/phpactor#1986]

**Comment by przepompownia** [comment]
With (n)vim it's very probable that `*` or `#` will do it immediately :wink:

**Comment by dantleech** [comment]
yeah, not something I've needed. But could be added in the PHPUnitExtension.

For me the more interesting part is somehow allowing `@dataProvider foobar` to be considered a method reference, so it s renamed etc, and/or auto generating a data provider - but snippets can kinda do that already in a better way. But maybe a good thing for Phpactor to provide that snippet.
